oracle-price-monitor/src/autotask.js

```diff
@@ -0,0 +1,270 @@
+const needle = require('needle');
+
+// import the DefenderRelayProvider to interact with its JSON-RPC endpoint
+const { DefenderRelayProvider } = require('defender-relay-client/lib/ethers');
+
+const TOKEN_ABI = [
+  'function decimals() view returns (uint8)',
+  'function symbol() view returns (string)',
+];
+const MAKER_TOKEN_ABI = [
+  'function decimals() view returns (uint256)',
+  'function symbol() view returns (bytes32)',
+];
+const CTOKEN_ABI = ['function underlying() view returns (address)'];
+
+const makerTokenAddress = '0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2'.toLowerCase();
+const saiTokenAddress = '0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359'.toLowerCase();
+const oddTokens = [makerTokenAddress, saiTokenAddress];
+
+const fortaApiEndpoint = 'https://api.forta.network/graphql';
+
+async function getDecimalsAndSymbol(cTokenAddress, provider) {
+  const cTokenContract = new ethers.Contract(
+    cTokenAddress,
+    CTOKEN_ABI,
+    provider,
+  );
+  const underlyingTokenAddress = await cTokenContract.underlying();
+
+  let decimals;
+  let symbol;
+  if (oddTokens.indexOf(underlyingTokenAddress.toLowerCase()) !== -1) {
+    const underlyingTokenContract = new ethers.Contract(
+      underlyingTokenAddress,
+      MAKER_TOKEN_ABI,
+      provider,
+    );
+
+    decimals = await underlyingTokenContract.decimals();
+    // need to convert decimals from uint256 to uint8
+    decimals = parseInt(decimals.toString(), 10);
+
+    symbol = await underlyingTokenContract.symbol();
+    // need to convert symbol from bytes32 to string
+    symbol = ethers.utils.parseBytes32String(symbol);
+  } else {
+    const underlyingTokenContract = new ethers.Contract(
+      underlyingTokenAddress,
+      TOKEN_ABI,
+      provider,
+    );
+    decimals = await underlyingTokenContract.decimals();
+    symbol = await underlyingTokenContract.symbol();
+  }
+  return { decimals, symbol };
+}
+
+function formatAmountString(amount, decimals) {
+  console.log("amount here", amount)
+  const amountBN = ethers.BigNumber.from(amount);
+  const divisorBN = ethers.BigNumber.from(10).pow(decimals);
+
+  // the ethers BigNumber implementation will discard the decimal
+  // portion of the value when we perform the division
+  let resultString = amountBN.toString();
+  if (resultString.length <= decimals) {
+    resultString = `0.${'0'.repeat(decimals - resultString.length)}${resultString[0]}`;
+  } else {
+    resultString = amountBN.div(divisorBN).toString();
+  }
+
+  // format the number to have comma separators for powers of 1000
+  const internationalNumberFormat = new Intl.NumberFormat('en-US');
+  return internationalNumberFormat.format(resultString);
+}
+
+async function createDiscordMessage(reporterPrice, cTokenAddress, transactionHash, provider) {
+  const { decimals, symbol } = await getDecimalsAndSymbol(cTokenAddress, provider);
+
+  const amountString = formatAmountString(reporterPrice, decimals);
+
+  console.log("amount string here", amountString)
```

Comment:
```suggestion
  console.log('amount string here', amountString);
```
Ref: https://github.com/airbnb/javascript#strings--quotes